clients/docs/ksql-datagen.rst

```diff
@@ -172,7 +171,6 @@ Consume Avro Records
 
       .. code-block:: bash
 
-         docker-compose exec connect bash -c 'kafka-avro-console-consumer --topic test2 --bootstrap-server $CONNECT_BOOTSTRAP_SERVERS --consumer-property ssl.endpoint.identification.algorithm=https --consumer-property sasl.mechanism=PLAIN --consumer-property security.protocol=SASL_SSL --consumer-property sasl.jaas.config="$SASL_JAAS_CONFIG_PROPERTY_FORMAT" --property basic.auth.credentials.source=$CONNECT_VALUE_CONVERTER_BASIC_AUTH_CREDENTIALS_SOURCE --property schema.registry.basic.auth.user.info=$CONNECT_VALUE_CONVERTER_SCHEMA_REGISTRY_BASIC_AUTH_USER_INFO --property schema.registry.url=$CONNECT_VALUE_CONVERTER_SCHEMA_REGISTRY_URL --max-messages 5'
```

Comment:
@ybyzek it's difficult to see how this removal is related to DEVX-2151, but if it's being removed, should the code-block directive above it be removed as well?

Reply:
@rspurgeon in trying to validate this PR against the ksql-datagen example, I discovered another issue (`KSQL-3812: remove log4j.properties setting for ksql-datagen`) that I rolled into this PR.  But it makes for a confusing PR & review process, so I will revert the change and submit in a new PR.